### ISSUE TITLE
Disable gosec temporarily in Nephio repo 

### ISSUE THREAD

**Issue by gvbalaji** [nephio-project/nephio#156]
We have multiple artifacts built in the Nephio  (https://github.com/nephio-project/nephio) repo. We have a go.mod file at the root and and one for each artifact . This is causing a few issues with gosec and the jobs are failing.  In order to unblock the team let us disable gosec jobs only on this repo for now. 

There will be [another issue](https://github.com/nephio-project/nephio/issues/157) created to dive deep into this and fix it the right way that involves some reorganization of code and prow jobs.

**Comment by radoslawc** [comment]
Sure it just needs removing it in repo's local file .prow.yaml. I've created PR for that. We'll revert that once that's sorted.
